cmd/root.go

```diff
@@ -79,3 +92,24 @@ func Execute() int {
 
 	return exitCode.value
 }
+
+var levelMap = map[string]log.Level{
```

Comment:
Should we move this logging stuff over to the `cmdutil` package?

Reply:
Done.